modules/backend/ServiceProvider.php

```diff
@@ -76,6 +76,7 @@ protected function registerAssetBundles()
             $combiner->registerBundle('~/modules/backend/formwidgets/fileupload/assets/less/fileupload.less');
             $combiner->registerBundle('~/modules/backend/formwidgets/nestedform/assets/less/nestedform.less');
             $combiner->registerBundle('~/modules/backend/formwidgets/richeditor/assets/js/build-plugins.js');
+            $combiner->registerBundle('~/modules/backend/formwidgets/colorpicker/assets/less/colorpicker.less');
```

Comment:
@w20k widget assets are recompiled when they're registered like this and then `php artisan october:util compile` is run.